### PR TITLE
Add modal to confirm if user really wants to delete game archives

### PR DESCRIPTION
**Pull request by xbozo**
Preview:
Screenshot_2

### CODE REVIEW

**Comment by xbozo** [comment]
Screenshot_2

File: src/renderer/pages/game-details/delete-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { Button, Modal } from "@renderer/components";
+import { useTranslation } from "react-i18next";
+import * as styles from "./delete-modal.css";
+
+type DeleteModalProps = {
```
Comment (hydralauncher) [inline src/renderer/pages/game-details/delete-modal.tsx:5]: Please use `interface` instead

File: src/renderer/pages/game-details/delete-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  visible: boolean;
+  title: string;
```
Comment (hydralauncher) [inline src/renderer/pages/game-details/delete-modal.tsx:7]: Why do you have to pass `title` and `description` as props if they're always the same and this component is going to be used in a single place?

File: src/renderer/pages/game-details/delete-modal.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            onClose();
+          }}
+          theme="outline"
+          disabled={deleting}
+        >
+          {t("delete")}
+        </Button>
+
+        <Button
+          onClick={() => {
```
Comment (hydralauncher) [inline src/renderer/pages/game-details/delete-modal.tsx:44]: Don't create a new function if not necessary, use `onClick={onClose}` instead

File: src/renderer/pages/game-details/delete-modal.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <Modal
+      visible={visible}
+      title={title}
+      description={description}
+      onClose={onClose}
+    >
+      <div className={styles.deleteActionsButtonsCtn}>
+        <Button
+          onClick={() => {
```
Comment (hydralauncher) [inline src/renderer/pages/game-details/delete-modal.tsx:33]: Move this to a separate function

**Review by hydralauncher — CHANGES_REQUESTED** [review]

**Comment by xbozo** [comment]
fix @hydralauncher

File: src/renderer/pages/game-details/delete-modal.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+export function DeleteModal({
+  onClose,
+  visible,
+  deleting,
+  deleteGame,
+}: DeleteModalProps) {
+  const { t } = useTranslation("game_details");
+
+  function handleDeleteGame() {
```
Comment (hydralauncher) [inline src/renderer/pages/game-details/delete-modal.tsx:20]: Use arrow functions within the component context

**Comment by hydralauncher** [comment]
LGTM

**Pull request merged**